src/recommender.py

```diff
@@ -366,9 +366,11 @@ class RecommendationTask:
 
     _analysis_name = 'recommendation_v2'
     description = 'Get Recommendation'
+    kronos_ecosystems = ['maven']
```

Comment:
For now, it is just one ecosystem per list, but for future we can use `set` instead.